controllers/utils.go

```diff
@@ -37,6 +38,24 @@ func IsOpenShift() (bool, error) {
 	return false, nil
 }
 
+func parseJobYAML(yamlData []byte) (*batchv1.Job, error) {
+	job := &batchv1.Job{}
+	err := yaml.Unmarshal(yamlData, job)
+	if err != nil {
+		return nil, err
+	}
+	return job, nil
+}
+
+func readJobYAML(jobFileName string) ([]byte, error) {
+	jobFilePath := filepath.Join(peerpodsImageJobsPathLocation, jobFileName)
+	yamlData, err := os.ReadFile(jobFilePath)
+	if err != nil {
+		return nil, err
+	}
+	return yamlData, nil
+}
+
```

Comment:
It looks like a `readYAML(path string, fileName string)` generic helper can be factored out.

Reply:
indeed, however, it's likely to be dropped in the future so i think we should leave it as is for now